Dockerfile.assisted-test-infra

```diff
@@ -18,14 +18,16 @@ RUN dnf -y install --enablerepo=crb \
   httpd-tools \
   jq \
   nss_wrapper \
-  python3 \
-  python3-devel \
   libvirt-client \
   libvirt-devel \
   libguestfs-tools \
   libxslt \
    && dnf clean all
 
+# Install latest available python version
+COPY --from=quay.io/eerez/python-builder:stream9-3.10.7 /python /python-installation
```

Comment:
just something I'm wondering if we can make it happen
is it possible to create ``quay.io/eerez/python-builder:stream9-3.10.7`` (or with some other name) as a pre-included stream9 based image that has python 3.10 installed?
it can save some time building images and be a little more agnostic to how we're getting those python3 dependencies

Reply:
for quite some time we wanted to try the base-images idea, so we might as well go along with it

Reply:
ACK
I can get along with it, but let's just fix some things first:
* Add that URL in a comment, so people will know where to find the source code
* That dir needs some cleanup, for example you have a ``__init__.py`` module with just the string ``FROM``
* Add a small README in that dir to make the cross-connect to the generated image in quay.io

Reply:
Sure, I just wanted to check first if it's working and get your opinion on the idea. I'll add those fixes.

Reply:
Updated https://github.com/eliorerz/tools/blob/master/python_builder/README.md